src/routers/statementRelevantOfficerRouter.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response, Router, NextFunction } from "express";
+import { StatementRelevantOfficerHandler } from "./handlers/role/statementRelevantOfficer";
+
+const router: Router = Router();
+const routeViews: string = "router_views/role";
+
+const handler = new StatementRelevantOfficerHandler();
+
+router.get("/statement-relevant-officer", async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        const viewData = await handler.execute(req, res);
+        res.render(`${routeViews}/statement-relevant-officer`, viewData);
+    } catch (err) {
+        res.status(500).send("Internal Server Error");
+    }
+});
+
+router.post("/statement-relevant-officer", async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        const selectedRole = req.body.WhatIsYourRole;
+
+        if (selectedRole === "director" || selectedRole === "general-partner" || selectedRole === "member-of-the-partnership" || selectedRole === "sole-trader") { // selectedRole equal to value base on HTML radio element
```

Comment:
The radio button seems to have only 2 values either director or someone else. But here other values are being checked as well. Are they required to be checked here?

Reply:
In this case, I checked all role option as the acceptance criteria covered all scenarios. However, depending on what was selected in the previous page the first option will changed, while 'I am someone else' remain constant.